### ISSUE TITLE
Comet tails hidden when comet nucleus is outside screen

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#1033]
This is a bug probably related to the solution to #389.

When comets are visible in the sky, their tails are hidden once the nucleus is off-screen. This is very prominent for the possible comet of the century, C2019 Y4 (ATLAS), which shows a tail of more than 60 degrees in late May. See these screenshots:

![stellarium-037](https://user-images.githubusercontent.com/37028368/78281803-88ecab80-751b-11ea-87eb-1f7716abeff1.png)

And with the nucleus outside of the screen (below the horizon in this example):
![stellarium-038](https://user-images.githubusercontent.com/37028368/78281861-a4f04d00-751b-11ea-9ab1-a0216003b11c.png)

Using 0.20 on a Windows 10 machine.

**Comment by gzotti** [comment]
Of course this is related. (sigh...) Solve one, open another.
But what absolute magnitude are you using? Is that related to (whose) reality? Or are you 7.9 magnitudes brighter than our standard solution?

**Comment by Atque** [comment]
@gzotti I have the exact same parameters as the default ssystem_minor.ini; absolute_magnitude = 7.9.

Some astronomers say the comet might become really bright, but mag. -17 in the evening sky at May 24 sounds ridiculous, and would probably make this the most brilliant comet since Homo sapiens evolved. :)

**Comment by gzotti** [comment]
Interesting. My Stellarium 0.20.0.1 gives mag 2.5, which is roughly what I *hope for*. I would definitely not believe mag -17 without solid (independent) evidence. Does your printed object information really print "Absolute magnitude: 7.90"?

**Comment by Atque** [comment]
@gzotti Yes, it does say "Absolute magnitude: 7.90". This is weird. I even renamed the _ssystem_minor.ini_ and restarted Stellarium, just to make sure Stellarium used the default. Still I get crazy values in late May. :/

**Comment by gzotti** [comment]
Sounds totally weird. I have no explanation for this.   =:-o 
Whoever is following this, the image above is NO indication for any super-bright comet... 
Do you have another computer to try? 
Was that download version 0.20.0 or 0.20.0.1? (@alex-w Not sure what the reason for the patch release was?)

**Comment by Atque** [comment]
@gzotti Yes, this is really, really strange. :) What magnitude to you get at, say 24 May at 00 UT? I use the 0.20.0 release. I'll try on the 0.20.0.1, but I doubt there's any difference.

I have no other computer to try on, but I have a different build (approx. 17740), and I don't even get that comet in the default file. :)

**Comment by gzotti** [comment]
Uh, great. I had added it manually. Our 0.20.0 includes a data set that shows a 

slope_parameter=21

This is nonsense. Please edit this and set it to 4.
